apps/showcase/src/app/app.component.ts

```diff
@@ -38,12 +38,14 @@ export class AppComponent implements OnDestroy {
   private subscriptions = new Subscription();
 
   constructor(router: Router, private offcanvasService: NgbOffcanvas) {
-    this.activeUrl$ = router.events.pipe(
-      filter((event): event is NavigationEnd => event instanceof NavigationEnd),
+    const onNavigationEnd$ = router.events.pipe(
+      filter((event): event is NavigationEnd => event instanceof NavigationEnd)
```

Comment:
nitpicking
```suggestion
      filter((event): event is NavigationEnd => event instanceof NavigationEnd),
      share()
```
to not have to execute the comparison everytime `onNavigationEnd$` emit